pkg/utils/utils_test.go

```diff
@@ -26,3 +29,31 @@ func TestLookupEnvWithDefaultUndefined(t *testing.T) {
 		t.Errorf("Expected %s=%s but got %s=%s", envKey, expected, envKey, res)
 	}
 }
+
+func TestCompareResources(t *testing.T) {
+	cases := [] struct {
+		current *v1.ResourceRequirements
+		desired *v1.ResourceRequirements
+		expected bool
+	} {
+		{
+			&v1.ResourceRequirements{},
+			&v1.ResourceRequirements{},
+			false,
+		},
+		{
+			&v1.ResourceRequirements{},
+			&v1.ResourceRequirements{
+				Limits: map[v1.ResourceName]resource.Quantity{"cpu": *resource.NewMilliQuantity(1000, resource.DecimalSI)},
```

Comment:
can we also add a test case with `Requests` ?

Reply:
Done